gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
Is this required ? As far as I can see The Jenkins Gradle plugin only provide Gradle runtime up to 2.12.

http://ci.xwiki.org/job/Contrib%20-%20Android%20Authenticator/ is currently failing because of that.